### ISSUE TITLE
Pointer offset question

### ISSUE THREAD

**Issue by alejandrodnm** [cfsamson/example-greenthreads#15]
Hello,

In:

https://github.com/cfsamson/example-greenthreads/blob/master/src/main.rs#L127-L128

Why do you use an offset of -32 and -24 and not -16 and -8? Like you did in:

https://cfsamson.gitbook.io/green-threads-explained-in-200-lines-of-rust/an-example-we-can-build-upon#running-our-example

Thanks for the books :)

**Comment by cfsamson** [comment]
Hi. Glad to hear you enjoy them.

This is a good question! The reason I initially used an offset of -32 and -24 was to keep as much of the code similar between Linux/Macos and Windows as possible.

The Windows section started out as a "bonus" paragraph, where I only changed the `switch` method but since there was so much to cover I ended up creating a separate Windows chapter after some time and I have forgotten to change the Linux/Macos implementation to use an offset of -16. 

I'll leave this issue open as a reminder to fix it on the next update of the book and the example repo.
